### DISCUSSION TITLE
Use a colored speaker icon for `AudioStreamPlayer`s

### DISCUSSION THREAD

**Discussion by mieldepoche** [godotengine/godot-proposals#9655] [Editor]
![image](https://github.com/godotengine/godot-proposals/assets/71937306/02c441a8-f2a4-48ec-b8e5-645bcbee4b97)
![image](https://github.com/godotengine/godot-proposals/assets/71937306/fb867238-9589-4cb1-92b9-d7289c532937)
![image](https://github.com/godotengine/godot-proposals/assets/71937306/b30e4ef1-1052-4604-8a46-9ba4e50867ec)

Those 3 icons only differ by a tiny color dot and I find them not symbolic enough to be pleasant to read.
As a bonus, the music notation used in the icon is, well, a *music* notation, not something related to sound in general.

Currently, the icon is made up of 2 parts:
1. the multi-colored note, this gradient having the "sound" meaning,
2. a small colored play icon, colored according to the spatialness of the node.

My point of view is the following:
- the gradient is useless. A good icon would convey the audio part much more naturally than the current design.
- the spatial part is way too small to be useful.

The solution I propose:
- Use an actual speaker icon instead
- Color it *fully* according to spatiality: either white for `AudioStreamPlayer`, blue for `AudioStreamPlayer2D` or red for `AudioStreamPlayer3D`
- Keep the gradient in `AudioStream`s since there is no common icon for them (some use the note, some don't)

![image](https://github.com/godotengine/godot-proposals/assets/71937306/cc6c7c93-8581-4a86-913a-797f14253f8d)

this is the same icon used by every other software I can think of:
blender|ubuntu|unity|godot, in the 3D view
-|-|-|-
![image](https://github.com/godotengine/godot-proposals/assets/71937306/dc2f4ecf-a449-4008-879a-11147c1fc5a2)|![image](https://github.com/godotengine/godot-proposals/assets/71937306/c9047d9b-a052-4c6a-8a00-5a3efd568e48)|![image](https://github.com/godotengine/godot-proposals/assets/71937306/3c2f7ad9-15c9-42ae-ad85-20921a3fae34)|![image](https://github.com/godotengine/godot-proposals/assets/71937306/308de9ab-864b-4bf2-bb86-d12e8508b65c)

**Comment by Calinou** [comment]
- Superseded by https://github.com/godotengine/godot-proposals/issues/9688.
